tsconfig.packages.json

```diff
@@ -17,6 +17,7 @@
       "@metamask/*": ["../*/src"]
     },
     "resolveJsonModule": true,
+    "skipLibCheck": true,
```

Comment:
Vitest seems to import non-existing types from Vite. This seems like the easiest way to avoid it, and actually speeds up building too.